src/services/ethereum/index.js

```diff
@@ -0,0 +1,73 @@
+// @flow
+
+import ethers from 'ethers';
+import ERC20ABI from './ERC20ABI.json';
+
+export default class EthereumService {
+  wallet: Object;
+  constructor(wallet: Object) {
+    this.wallet = wallet;
+  }
+  async getBalance(): Promise<void> {
+    const balance = await this.wallet.getBalance('latest');
+    return balance;
+  }
+
+  async getTokenBalance(tokenAddress: string): Promise<void> {
+    const abi = ERC20ABI;
+    const contract = new ethers.Contract(tokenAddress, abi, this.wallet.provider);
+    const balance = await contract.balanceOf(this.wallet.address);
+    return balance;
+  }
+
+  async sendTokens(tokenAddress: string, toAddress: string, tokenAmount: string) {
+    const abi = ERC20ABI;
+    const contract = new ethers.Contract(tokenAddress, abi, this.wallet);
+    const formattedTokenAmount = ethers.utils.parseUnits(tokenAmount, 18);
+    const overrideOptions = {
+      gasLimit: 1000000,
+    };
+    const transactionHash = await contract.transfer(toAddress, formattedTokenAmount, overrideOptions);
+    return transactionHash;
+  }
+
+  async estimateGas(gasPrice: string, data: string): Promise<void> {
+    const transaction = {
+      gasPrice: ethers.utils.bigNumberify(gasPrice),
+      to: '0xF0D346A86A68086846363185d24D5893F4353A78',
```

Comment:
What is that address about?

Reply:
It is just a dummy address to help create a dummy transaction. It doesn't affect the function, it just needs to have a value